src/coreclr/gc/gc.cpp

```diff
@@ -48186,6 +48186,42 @@ HRESULT GCHeap::Initialize()
     {
         return E_OUTOFMEMORY;
     }
+
+    /*
+     * Allocation requests less than loh_size_threshold will be allocated on the small object heap.
+     * For regions, we constrainted that objects must stay on a single region, so it is impossible to allocate
```

Comment:
that's the same for segments too. it's just segments are a lot larger. as we talked about, we should decide if we want to have a different size for segments. I'm ok with having the same value for both so users don't need to concern themselves with whether they are using segments or regions.

Reply:
oh and please remember to update the documentation for this config. thanks!

Reply:
Here (https://github.com/dotnet/docs/pull/38763) is the documentation update PR.